easybuild/easyconfigs/f/FreeFem++/FreeFem++-3.58-foss-2017b-downloaded-deps.eb

```diff
@@ -1,4 +1,4 @@
-easyblock = 'ConfigureMake'
+easyblock = 'EB_FreeFem'
```

Comment:
This is not needed if you fix the class name for the easyblock in https://github.com/easybuilders/easybuild-easyblocks/pull/1488